README.md

```diff
@@ -4,7 +4,7 @@
 
 ## What is Rafiki?
 
-Rafiki is open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
+Rafiki is open source software that provides an efficient solution for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```

Comment:
Nit-picky, but Account Servicing Entity should be in lowercase.

Reply:
I think we need a convention here, I would have argued "Account Servicing Entity" is always capitalized.